portal/src/texts/profile/tone.md

```diff
@@ -42,15 +39,49 @@ Vi tar kunden på alvor, sørger for at de ikke føler seg dumme.
 Vi skryter ikke av oss selv, men lar kundene fortelle om sine opplevelser.
 
 ### Presis
-
-Vi skal bli verdensmestere i klarspråk
-Vi snakker norsk, og bruker riktig grammatikk og språkregler.
+__Vi skal bli gode i klarspråk__
+Vi snakker og skriver norsk, og bruker riktig grammatikk og språkregler.
 
 Vi er presise i alt vi sier, uansett om det gjelder hva forsikringen dekker, hvorfor vi samler inn data om kundene våre, eller hvordan vi gir kundene oversikt og full kontroll over forsikringene sine.
 
 Kundene skal oppleve at de får informasjon tilpasset akkurat seg, som er relevant, kommer til riktig tid og i riktig kanal.
 
 Eksempler i kommunikasjonen:
-Vi bruker «Ti tips for å skrive bedre».
+Vi følger gode skriveråd, for eksempel «Ti tips for å skrive bedre».
 Vi fjerner stammespråk.
 Vi hjelper hverandre med å lese korrektur på alle tekster.
+
+## Skriveråd
+1.	_Hvem skriver du for?_
+ -	Vurder hvem som er målgruppen din. 
+ -	Hvor skal mottakeren lese teksten? På en nettside eller et digitalt skjema?
+ 
+2.	_Hva er målet med teksten?_
+- Gjør budskapet tydelig fra begynnelsen av. 
+-	Tenk på hva mottakeren vet fra før. Ta bare med det som er nødvendig.
```

Comment:
Her må vi nok omformatere litt for at det skal blir riktig struktur på koden som spyttes ut (husk at vi kun definerer semantisk struktur, og ikke layout, i markdown-filene):

```markdown
1. _Hvem skriver du for?_
    - Vurder hvem som...
    - Hvor skal mottakeren...
2. _Hva er målet med teksten?_
    - Gjør budskapet tydelig...
```
Hvis stegene ikke skal være en nummerert liste ville jeg gjort dem om til overskrifter på et lavt nivå:
```markdown
### 1. _Hvem skriver du for?_
```